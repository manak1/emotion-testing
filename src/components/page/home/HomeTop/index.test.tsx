import { composeStories } from "@storybook/testing-react"

import { render } from "~/test/test-utils"

import * as stories from "./index.stories"
const { Default } = composeStories(stories)

describe("HomeTopのテスト", () => {
  it("snapshot", () => {
    const { asFragment } = render(<Default />)
    expect(asFragment()).toMatchSnapshot()
  })
})
