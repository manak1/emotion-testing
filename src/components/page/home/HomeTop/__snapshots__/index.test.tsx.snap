// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeTopのテスト snapshot 1`] = `
<DocumentFragment>
  .emotion-0 {
  font-weight: bold;
  margin: 0;
  font-size: 32px;
  text-align: center;
}

<h1
    class="emotion-0"
  >
    制度作ったったー
  </h1>
  .emotion-0 {
  font-size: 14px;
}

<p
    class="emotion-0"
  >
    こんな制度あったらいいな！を叶える事が出来るサイトです。もしかしたら本当に採用されちゃうかも..!?
  </p>
  .emotion-0 {
  min-width: 36px;
  outline: none;
  padding: 10px 16px;
  border: none;
  border-radius: 8px;
  cursor: pointer;
  font-size: 16px;
  width: 100%;
  font-weight: bold;
  background-color: #7856FF;
  color: #fff;
  -webkit-transition: box-shadow,filter,0.2s ease-in;
  transition: box-shadow,filter,0.2s ease-in;
  padding: 10px 16px;
}

.emotion-0:hover {
  -webkit-filter: brightness(95%);
  filter: brightness(95%);
  -webkit-transition: filter 0.2s ease-out;
  transition: filter 0.2s ease-out;
}

.emotion-0:focus {
  box-shadow: 0 0 0 2px rgba(120, 86, 255, 0.3);
  -webkit-transition: box-shadow 0.2s ease-out;
  transition: box-shadow 0.2s ease-out;
}

<button
    class="emotion-0"
  >
    制度をつくる
  </button>
</DocumentFragment>
`;
